info.limpet.rcp/src/info/limpet/rcp/data_provider/data/CollectionWrapper.java

```diff
@@ -18,42 +18,6 @@ public CollectionWrapper(final LimpetWrapper parent,
 		_collection = collection;
 	}
 
-	
```

Comment:
I removed these methods intentionally. I think that wrappers equality should be based on the wrapped object + parent. Since a model object might appear in multiple places in the tree, with the current approach it won't be possible to select certain nodes in the tree (JFace viewer will pickup the first appearance I believe). But then yes - removing hashCode and equals broke the "preserve selection on refresh" functionality.